docs/en/creating-mfa-method-frontend.md

```diff
@@ -0,0 +1,218 @@
+# Creating a new MFA method: Frontend
+
+## Introduction
+
+The MFA module provides a clear path for creating additional authentication methods. In this document we'll cover how to
+implement the front-end portion of the required code, using the Basic Math method as an example. Some prior experience
+with React / Redux is recommended.
+
+The front-end components of MFA make use of `react-injector` (Injector) to allow sharing of React components and Redux
```

Comment:
Do you want to link to the repo for the library here? Assuming the documentation in that repo will improve going forward.

Reply:
```suggestion
The front-end components of MFA make use of [`react-injector`](https://github.com/silverstripe/react-injector/) (Injector) to allow sharing of React components and Redux
```